pandas/io/pytables.py

```diff
@@ -1269,18 +1269,22 @@ def walk(self, where="/"):
 
             yield (g._v_pathname.rstrip("/"), groups, leaves)
 
-    def get_node(self, key: str):
+    def get_node(self, key: str) -> Optional["Node"]:
         """ return the node with the key or None if it does not exist """
         self._check_if_open()
         if not key.startswith("/"):
             key = "/" + key
 
         assert self._handle is not None
         try:
-            return self._handle.get_node(self.root, key)
+            node = self._handle.get_node(self.root, key)
         except _table_mod.exceptions.NoSuchNodeError:  # type: ignore
             return None
 
+        assert _table_mod is not None  # for mypy
```

Comment:
if you move this above the try/except, you can remove the ignore from a previous commit?

Reply:
yep, good idea